lib/capybara.rb

```diff
@@ -82,6 +82,7 @@ class << self
     # [reuse_server = Boolean]  Reuse the server thread between multiple sessions using the same app object (Default: true)
     # [threadsafe = Boolean]  Whether sessions can be configured individually (Default: false)
     # [server = Symbol]  The name of the registered server to use when running the app under test (Default: :webrick)
+    # [default_node_set_options = Hash]  The default options passed to Node::set (Default: {})
```

Comment:
Please change this back to `default_set_options` - The only public `set` API is on Node, so there's not need to specify 'node' in the name